rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -5625,4 +5785,60 @@ private Transaction getMockedRskTxWithHash(String s) {
         byte[] hash = Keccak256Helper.keccak256(s);
         return new SimpleRskTransaction(hash);
     }
+
+    private BridgeSupport getBridgeSupportMockForReceiveHeader(
+            BtcBlock btcBlock,
+            BtcBlockStoreWithCache btcBlockStore,
+            BridgeStorageProvider provider,
+            StoredBlock storedBlock,
+            ActivationConfig.ForBlock activation
+    ) throws BlockStoreException {
+        return getBridgeSupportMockForReceiveHeader(
+                btcBlock,
+                btcBlockStore,
+                provider,
+                storedBlock,
+                mock(Block.class),
+                activation
+        );
+    }
+
+    private BridgeSupport getBridgeSupportMockForReceiveHeader(
```

Comment:
BridgeSupport is not really a mock, you are configuring it for a given test, consider renaming the methods
```suggestion
    private BridgeSupport getBridgeSupportConfiguredToTestReceiveHeader(
```